File: sdks/typescript/examples/market-data/orderbook.ts
import pmxt from 'pmxtjs';

const main = async () => {
    const api = new pmxt.Kalshi();
    const markets = await api.fetchMarkets({ slug: 'KXFEDCHAIRNOM-29' });
    const warsh = markets.find(m => m.outcomes[0]?.label === 'Kevin Warsh');

    if (warsh) {
        const book = await api.fetchOrderBook(warsh.outcomes[0].outcomeId);
        console.log(book);
    } else {
        console.log('Market not found');
    }
};

main();

File: sdks/python/examples/market-data/orderbook.py
import pmxt

def main():
    api = pmxt.Kalshi()
    markets = api.fetch_markets(slug='KXFEDCHAIRNOM-29')
    warsh = next((m for m in markets if m.outcomes[0].label == 'Kevin Warsh'), None)

    if warsh:
        book = api.fetch_order_book(warsh.outcomes[0].outcome_id)
        print(book)
    else:
        print("Market not found")

if __name__ == "__main__":
    main()
